src/main/java/be/uantwerpen/minelabs/gui/charged_point_gui/ChargedPointBlockScreenHandler.java

```diff
@@ -0,0 +1,88 @@
+package be.uantwerpen.minelabs.gui.charged_point_gui;
+
+import be.uantwerpen.minelabs.gui.ScreenHandlers;
+import be.uantwerpen.minelabs.inventory.slot.FilteredSlot;
+import be.uantwerpen.minelabs.util.Tags;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.entity.player.PlayerInventory;
+import net.minecraft.inventory.Inventory;
+import net.minecraft.inventory.SimpleInventory;
+import net.minecraft.item.ItemStack;
+import net.minecraft.screen.ScreenHandler;
+import net.minecraft.screen.slot.Slot;
+import org.jetbrains.annotations.NotNull;
+
+public class ChargedPointBlockScreenHandler extends ScreenHandler {
+
+    private final Inventory inventory;
+
+    public ChargedPointBlockScreenHandler(int syncId, PlayerInventory inventory) {
+        this(syncId, inventory, new SimpleInventory(1));
+    }
+
+    public ChargedPointBlockScreenHandler(int syncId, @NotNull PlayerInventory playerInventory, Inventory inventory) {
+        super(ScreenHandlers.CHARGED_POINT_SCREEN_HANDLER, syncId);
+        checkSize(inventory, 1);
+        this.inventory = inventory;
+        inventory.onOpen(playerInventory.player);
+
+        //TODO CORRECT COORDS SLOT
```

Comment:
Are they correct now? Remove comment then.

Reply:
Incase Art decided on a different position for the item.